ddtrace/profiling/collector/memalloc.py

```diff
@@ -182,8 +179,8 @@ def collect(self):
                     # DEV: This might happen if the memalloc sofile is unlinked and relinked without module
                     #      re-initialization.
                     LOG.debug("Invalid state detected in memalloc module, suppressing profile")
-
-        if self._export_py_enabled:
+            return []
```

Comment:
```suggestion
            return tuple()
```
for type consistency?

Reply:
I think that's right.  Yoink!